pyinfra/facts/win_files.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import unicode_literals
```

Comment:
This file should be called `windows_files.py`.

Reply:
Ok. Done.